src/event_loop.rs

```diff
@@ -430,10 +432,16 @@ impl rwh_06::HasDisplayHandle for dyn ActiveEventLoop + '_ {
 ///
 /// - A zero-sized type that is likely optimized out.
 /// - A reference-counted pointer to the underlying type.
-#[derive(Clone, PartialEq, Eq)]
```

Comment:
Equality comparisons may be worth keeping here.

Reply:
I'd assume via display handle direct comparison? Since I don't see implementing it otherwise.

Reply:
I've brought it back, so it's a bit different now, since we have an error case. I'm not sure why this is needed though, given that the underlying handle is unique.